src/ontology/mondo-ingest.Makefile

```diff
@@ -335,7 +335,7 @@ build-mondo-ingest:
 	$(MAKE) refresh-imports exclusions-all slurp-all mappings matches \
```

Comment:
are all production data releases generated with this command?

Reply:
All of them! `build-mondo-ingest` is the only command I ever run to update the data.

Reply:
Ok, so is `$(MAKE_FAST)` then just used for development work where the imports don't need to be refreshed for each iteration/ingest run?

Reply:
Exactly!